guides/repo-view.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: doc
+sublayout: guide
+title: Working with repositories in the UI
```

Comment:
Well, it doesn't explain working with repos on the command line. Not sure whether there is a better description